runner/entrypoint.sh

```diff
@@ -24,6 +24,16 @@ if [ ! -z "${STARTUP_DELAY_IN_SECONDS}" ]; then
   sleep ${STARTUP_DELAY_IN_SECONDS}
 fi
 
+if [ "${DOCKER_ENABLED}" == "true" ] && [ "${DISABLE_WAIT_FOR_DOCKER}" != "true" ]; then
+  log "Docker enabled runner detected and Docker daemon wait is enabled"
```

Comment:
```suggestion
  log "Docker is enabled and Docker daemon wait is enabled"
```

Reply:
This could be worded better tbf, I'll make the change